third_party/terraform/tests/resource_cloudbuild_trigger_test.go

```diff
@@ -151,6 +150,19 @@ resource "google_cloudbuild_trigger" "build_trigger" {
       args = ["build", "-t", "gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA", "-f", "Dockerfile", "."]
       timeout = "300s"
     }
+    artifacts {
+      images = ["gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA"]
+      objects {
+        timing {
```

Comment:
Isn't this field output-only? There's plenty of other fields that are settable that would be good to see in a test :)